state-chain/pallets/cf-broadcast/src/migrations/initialize_broadcast_timeout_storage.rs

```diff
@@ -0,0 +1,84 @@
+use cf_primitives::{BlockNumber, MILLISECONDS_PER_BLOCK};
+use frame_support::{traits::OnRuntimeUpgrade, weights::Weight};
+use old::try_get_timeout_for_type;
+
+use crate::*;
+
+// MINUTES constant copied from `runtime/src/constants.rs`,
+// in order to use same timeout values as given in `node/src/chain_spec.rs`
+const MINUTES: BlockNumber = 60_000 / (MILLISECONDS_PER_BLOCK as BlockNumber);
+
+mod old {
+	use cf_primitives::BlockNumber;
+
+	use super::*;
+
+	// Same timeout values as previously defined in `#[pallet::constant]`s
+	// and same as currently used in `node/src/chain_spec.rs`
+	pub const ETHEREUM_BROADCAST_TIMEOUT: BlockNumber = 5 * MINUTES;
+	pub const POLKADOT_BROADCAST_TIMEOUT: BlockNumber = 4 * MINUTES;
+	pub const BITCOIN_BROADCAST_TIMEOUT: BlockNumber = 90 * MINUTES;
+	pub const ARBITRUM_BROADCAST_TIMEOUT: BlockNumber = 2 * MINUTES;
+	pub const SOLANA_BROADCAST_TIMEOUT: BlockNumber = 4 * MINUTES;
+
+	// For testing purposes we also have to set the timeout for the mock configuration,
+	// following `BROADCAST_EXPIRY_BLOCKS` in `mock.rs`
+	pub const MOCK_ETHEREUM_BROADCAST_TIMEOUT: BlockNumber = 4; //
+
+	pub fn try_get_timeout_for_type<T: Config<I>, I: 'static>() -> Option<BlockNumberFor<T>> {
```

Comment:
nit: `try` tends to imply `Result` 
```suggestion
	pub fn maybe_get_timeout_for_type<T: Config<I>, I: 'static>() -> Option<BlockNumberFor<T>> {
```